src/Admin/test/VMware.ScriptRuntimeService.AdminEngine.Tests/VCTrustedCertificatesCollectorTests.cs

```diff
@@ -0,0 +1,235 @@
+                Content = new StringContent(@"{
+   ""cert_chain"": {
+      ""cert_chain"": [
+        ""-----BEGIN CERTIFICATE-----\nMIIFOzCCA6OgAwIBAgIJAOrjMkQMNfJUMA0GCSqGSIb3DQEBCwUAMIGoMQswCQYD\nVQQDDAJDQTEXMBUGCgmSJomT8ixkARkWB3ZzcGhlcmUxFTATBgoJkiaJk/IsZAEZ\nFgVsb2NhbDELMAkGA1UEBhMCVVMxEzARBgNVBAgMCkNhbGlmb3JuaWExKjAoBgNV\nBAoMIXNjMi0xMC0xODUtMjI1LTE5OS5lbmcudm13YXJlLmNvbTEbMBkGA1UECwwS\nVk13YXJlIEVuZ2luZWVyaW5nMB4XDTIzMDIxMDEyMDAzNVoXDTMzMDIwNzEyMDAz\nNVowgagxCzAJBgNVBAMMAkNBMRcwFQYKCZImiZPyLGQBGRYHdnNwaGVyZTEVMBMG\nCgmSJomT8ixkARkWBWxvY2FsMQswCQYDVQQGEwJVUzETMBEGA1UECAwKQ2FsaWZv\ncm5pYTEqMCgGA1UECgwhc2MyLTEwLTE4NS0yMjUtMTk5LmVuZy52bXdhcmUuY29t\nMRswGQYDVQQLDBJWTXdhcmUgRW5naW5lZXJpbmcwggGiMA0GCSqGSIb3DQEBAQUA\nA4IBjwAwggGKAoIBgQDAyDKYVYwiCMZDSAgxHu595pU/W5pVFMv4gA01z2XfrOZY\nSuIQFY2PTtxjMM7p/3+oPw/sWXhO6ig94jguBZVselu/ksW+N2U+2nUJoScdGGio\nfGvXsV0DmJ0nMfdnSpVmY6R1ImIqZkRjonR4qpHGhzRjqjzSL/hU1Djiez+pIaKW\nugQ9QCFO+DVwiZ2OQqNESPOPQHera0bn4qzJSAjjibsyaoI7MCqhC0cM3oIZ8xLK\nq4QoP0fhUlv8fGLSb3gztO9QD3GGGZf/3nb3Xkf4l1TszKljzVMEjJbztnTiNIHj\nkRd1Am0X3DWpQl2yI6qwOU/Jbt7gVVad+7c9lmxnHVrMjX08pNF6A5k+X7MPg3CS\nWVcfs2LX/J41bTfOQpCgWYZnsh7OfSVlIDthfwh3f8MvSHc+JkACCJwVHTxR4uNW\nER48InN6p2/V+1GglliV/fo9ZMW9e6fI5EwQbEBGScOvpzkV59ABsl4SE0M7J8WT\n3AFrg+hO/cZkQACuTL0CAwEAAaNmMGQwHQYDVR0OBBYEFLobb584cG5Pqq+KoKFq\nfh5gxOv+MB8GA1UdEQQYMBaBDmVtYWlsQGFjbWUuY29thwR/AAABMA4GA1UdDwEB\n/wQEAwIBBjASBgNVHRMBAf8ECDAGAQH/AgEAMA0GCSqGSIb3DQEBCwUAA4IBgQCA\nynTFEvpjnBcrV7gjI2GBbVwK4kJqRURWbFMUZO/pgUKgT9pA7f2bOuoVgnLn0suY\nXXTPK0hBRQ5CsPvgUPuq2B3u9rOf25WMlnasDWqG3NPi+8GFsYKRr2nQecClI36q\nPmafcQ+cao3fm5OEXrecZRb5tG313p0YtlYgu9dJRjIw9CcJHrDsNKvj5WMyqrD0\ndHWlzq+DvZSZRLbk+ddn1QaU9D6SXYY+t8rSH+XvFj+RCgNZ1zWi4oxtx22rfMW6\n2dDooR4tRZ4FdtD9OOBTUg8bZs78hgBHD+2Hi5pH+HobO8yYvPs1BHPiAL/7/GCi\nRviQ205ZFw0hbMn8PMOQrHilFA1jx4NxqRlEWSBSOFVrmuB2YftHit77rzwfvHCl\nZ27QNsJu6u+wbGb9IMyipNsWWZ8sr7dAegdm3JkgJ9Ig89BfxsIfGXykuKrVYyx3\nrbhtJpljRHRdB0in5pvu36XCTrUWKcYoEjPSPrAr8MEd6hRqx1AlHglo41lQvNM=\n-----END CERTIFICATE-----\n-----BEGIN X509 CRL-----\nMIICpDCCAQwCAQEwDQYJKoZIhvcNAQELBQAwgagxCzAJBgNVBAMMAkNBMRcwFQYK\nCZImiZPyLGQBGRYHdnNwaGVyZTEVMBMGCgmSJomT8ixkARkWBWxvY2FsMQswCQYD\nVQQGEwJVUzETMBEGA1UECAwKQ2FsaWZvcm5pYTEqMCgGA1UECgwhc2MyLTEwLTE4\nNS0yMjUtMTk5LmVuZy52bXdhcmUuY29tMRswGQYDVQQLDBJWTXdhcmUgRW5naW5l\nZXJpbmcXDTIzMDIxNTEzNTAzNVoXDTIzMDMxNzEzNTAzNVqgLzAtMAoGA1UdFAQD\nAgELMB8GA1UdIwQYMBaAFLobb584cG5Pqq+KoKFqfh5gxOv+MA0GCSqGSIb3DQEB\nCwUAA4IBgQAGIImskOv8S2F140sGtP/mZ5MossDuks1cfbAC6nwsrPBDcwaoqpnl\n317ixIb3BHoW0b3MQydYivzybYehp//HxWOzqFFu0DJtBxEYbFADbxR33mr2O2hu\n8ANT8zTUgosBlheLqUZSH5DumAtiZa/Q1JKUGCG4QuFoMHi5EPfRK5/qdFeOEz+I\n1OELahB+9pyd6kO8EhzmmhN8UFUwImGJhdAR0NdXE7naBEQv2Y/nKhOk8gbJUEAR\nifDNeVdptKQR0cp+rS7ygscU1tfIntzdd9hkVGpSwuPkOK27iSqxjO9OQW6/x57M\nbJc1t39KmbqiwuDf/anJvzyM4sUVPPw4+ZMZdoqkBPW58eRZdk+x5I6kXQf+ZntC\nEvcSdV0JQGC8mR+j+YD36lfLIkL60payKMwohIhEr7Cj2SMVEJRZv3RgHaWi+e6U\nKkgOV2RbFt5wV5A+CczVJVDKPU4Q0wFsV23HIhURXmEDUz1ZLZ6I2vJaj5eJ80za\nYSCgKPcAsxA=\n-----END X509 CRL-----""
```

Comment:
Add a test where the chain contains more than 1 entry.
Add a test where the chain is not formatted correctly